pyperformance/data-files/benchmarks/bm_yaml/run_benchmark.py

```diff
@@ -0,0 +1,91 @@
+"""
+Script for testing the performance of YAML parsing, using yaml.
+
+This will dump/load several real world-representative objects a few thousand
+times. The methodology below was chosen for was chosen to be similar to
+real-world scenarios which operate on single objects at a time.
```

Comment:
```suggestion
This will dump/load several real world-representative objects a few thousand
times. The methodology below was chosen  to be similar to
real-world scenarios which operate on single objects at a time.
```